randovania/gui/ui_files/preset_dock_rando.ui

```diff
@@ -103,7 +92,7 @@
            <item row="5" column="0" colspan="2">
             <widget class="QLabel" name="label">
              <property name="text">
-              <string>Only Vanilla is compatible with multiworld sessions.</string>
+              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The full door lock mode is &lt;span style=&quot; font-weight:700;&quot;&gt;not&lt;/span&gt; compatible with multiworld sessions.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
this is both the opposite of the truth, and is using the old name lol

Reply:
there's a bolded "not" in the escaped html mess there.

Reply:
this is why I've been slowly changing every label I touch to markdown lmao